src/test/crimson/test_socket.cc

```diff
@@ -0,0 +1,202 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#include <seastar/core/app-template.hh>
+#include <seastar/core/gate.hh>
+#include <seastar/core/sharded.hh>
+#include <seastar/core/sleep.hh>
+
+#include "crimson/common/log.h"
+#include "crimson/net/Errors.h"
+#include "crimson/net/Fwd.h"
+#include "crimson/net/Socket.h"
+
+namespace {
+
+using seastar::future;
+using ceph::net::error;
+using ceph::net::Socket;
+using ceph::net::SocketFRef;
+
+seastar::logger& logger() {
+  return ceph::get_logger(ceph_subsys_ms);
```

Comment:
i'd suggest use a dedicated `seastar::logger` for testing instead of reusing the one in crimson.

Reply:
changed to use logger named "test".